File: 01-java-moderno/ClubVip.java
record Persona(String nombre, int edad) {}

List<Persona> personas = List.of(new Persona("ana", 17), new Persona("beto", 20), new Persona("carla", 25));

List<String> nombresVip = personas
                       .stream()
                       .filter(p -> p.edad() >= 18)
                       .map(p -> p.nombre().toUpperCase())
                       .toList();

File: 02-js-moderno/clubVip.js
const personas = [{nombre: "ana", edad: 17}, {nombre: "beto", edad: 20}, {nombre: "carla", edad: 25}];

const personasMayorEdad = personas
                    .filter(p => p.edad >= 18)
                    .map(p => `Mayor de edad ${p.nombre.toUpperCase()}`)
console.log(personasMayorEdad);